posts/2021-04-12-async-vision-doc-shiny-future.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title: "Brainstorming Async Rust's Shiny Future"
+author: Niko Matsakis
+description: "Brainstorming Async Rust's Shiny Future"
+team: the Async Foundations Working Group <https://rust-lang.github.io/wg-async-foundations/>
+---
+
+On March 18th, we [announced the start of the Async Vision Doc][announce] process. Since then, we've landed [17 "status quo" stories][sq] and we have [5 more stories in open PRs][prs]; [Ryan Levick] and [I] have also hosted more than ten [collaborative][] [writing][] [sessions][] over the course of the last few weeks.
+
+[collaborative]: https://smallcultfollowing.com/babysteps/blog/2021/03/22/async-vision-doc-writing-sessions/
+[writing]: https://smallcultfollowing.com/babysteps/blog/2021/03/25/async-vision-doc-writing-sessions-ii/
+[sessions]: https://smallcultfollowing.com/babysteps/blog/2021/03/29/async-vision-doc-writing-sessions-iii/
+
+Now that we have a good base of "status quo" stories, we are starting to imagine what the [✨ "shiny future" ✨][sf] might look like. **We want your help!** If you have a great idea for Async Rust[^youknow], then [take a look at the template and open a PR][template]! Alternatively, if you have an idea for a story but would like to discuss it before writing, you can [open a "shiny future" issue][open]. Also, we would still love to get more ["status quo" stories][sq], so please continue to share those.
```

Comment:
I think we should clarify (by copying, not linking) what "Async Rust" here refers to. Particularly for the shiny future stories, it's going to be key that we want (and I genuinely don't know, so just inventing here) the experience to be just like with status quo, inclusive of both ecosystem and fundamentals in std.

Do we expect to have an example or two ready by the announcement on Monday? Maybe having a few would help people feel that they're ideas definitely are welcome.